BitwardenShared/Core/Platform/Services/ReviewPromptService.swift

```diff
@@ -6,11 +6,20 @@ import Foundation
 /// A protocol for a `ReviewPromptService` which determines if a user is eligible for a review prompt.
 ///
 protocol ReviewPromptService {
+    /// Clears all tracked user actions.
+    ///
+    func clearUserActions() async
```

Comment:
🤔 Should this be called when user logs out? Maybe it's planned for a different PR but wanted to ask just in case.

Reply:
We are not tracking actions on a per-user basis; instead, we are tracking user actions across the entire app. If multiple users are logged in, all user actions will be aggregated and counted collectively.